src/components/mods/maps/mapsTable.tsx

```diff
@@ -202,37 +201,144 @@ const MapsTable = (
                 },
                 {
                     accessor: "qualityName",
                     title: "Quality",
+                    ellipsis: true,
+                    render: (mapWithTechAndRatingInfo) => {
+                        if (mapWithTechAndRatingInfo.qualityCount === 0) return (
+                            <Text
+                                size="sm"
+                            >
+                                {mapWithTechAndRatingInfo.qualityName}
+                            </Text>
+                        );
+
+                        return (
+                            <ModsTableTooltip
+                                targetString={mapWithTechAndRatingInfo.qualityName}
+                                dropdownString={`${mapWithTechAndRatingInfo.qualityName}: ${mapWithTechAndRatingInfo.qualityCount} ratings`}
+                            />
+                        );
+                    },
                     cellsClassName: classes.columnCells,
                 },
                 {
                     accessor: "difficultyName",
                     title: "Difficulty",
+                    ellipsis: true,
+                    render: (mapWithTechAndRatingInfo) => {
+                        const difficultyNameFromMap = mapWithTechAndRatingInfo.difficultyName;
+
+                        let difficultyStringForDisplay: string;
+                        if (mapWithTechAndRatingInfo.difficultyCount === 0) {
+                            difficultyStringForDisplay = mapWithTechAndRatingInfo.difficultyName;
+                        } else {
+                            const [parentDifficulty, childDifficulty] = difficultyNameFromMap.split(": ");
+
+                            if (parentDifficulty === undefined || childDifficulty === undefined) return "";
+
+                            difficultyStringForDisplay = `${childDifficulty} ${parentDifficulty}`;
+                        }
+
+
+                        if (mapWithTechAndRatingInfo.difficultyCount === 0) return (
+                            <Text
+                                size="sm"
+                            >
+                                {mapWithTechAndRatingInfo.difficultyName}
+                            </Text>
+                        );
+
+                        return (
+                            <ModsTableTooltip
+                                targetString={difficultyStringForDisplay}
+                                dropdownString={`${difficultyStringForDisplay}: ${mapWithTechAndRatingInfo.difficultyCount} ratings`}
+                            />
+                        );
+                    },
                     cellsClassName: classes.columnCells,
                 },
                 {
                     accessor: "lengthName",
                     title: "Length",
+                    ellipsis: true,
                     cellsClassName: classes.columnCells,
```

Comment:
No render function here - is this intentional?

Reply:
It was, but now that you point it out, I'm going to update this to also render the length description in a tooltip.